### PR TITLE
feat(css_parser): support `@font-palette-values`

### LINKED ISSUE #1262: CSS Parser: parse font palette values at rule
https://developer.mozilla.org/en-US/docs/Web/CSS/@font-palette-values

### PR DESCRIPTION
**Pull request by unvalley**
## Summary

Closes #1262

This PR makes css parser parse `@font-palette-values`.

## Test Plan

Generated css grammar and formatter.

Added snapshot tests

### CODE REVIEW

File: crates/biome_css_parser/tests/css_test_suite/ok/at_rule/at_rule_font_palette_values.css
Comment (unvalley) [inline crates/biome_css_parser/tests/css_test_suite/ok/at_rule/at_rule_font_palette_values.css:1]: @denbezrukov 

This implementation of `@font-palette-values` is probably almost the same syntax as `@counter-style` and mimics it. But when I run the snapshot test, it fails with the following error. If you have any insight into this, please let me know.

```
at_rule_font_palette_values.css:1:2 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Unexpected value or character.
  
  > 1 │ @font-palette-values ident {  }
      │  ^^^^^^^^^^^^^^^^^^^
    2 │ 
  
  i Expected one of:
  
  - charset
  - color-profile
  - container
  - counter-style
  - document
  - font-face
  - font-feature-values
  - font-palette-values
  - import
  - keyframes
  - layer
  - media
  - namespace
  - page
  - property
  - supports
  - viewport
  - scope
```

Comment (denbezrukov) [reply crates/biome_css_parser/tests/css_test_suite/ok/at_rule/at_rule_font_palette_values.css:1]: I guess that we have to increase the limit up to 22 to fit `font-palette-values`
https://github.com/biomejs/biome/blob/efc084ce1355c3de4c6104c3beff61abb8ce82d1/crates/biome_css_parser/src/lexer/mod.rs#L725

Comment (unvalley) [reply crates/biome_css_parser/tests/css_test_suite/ok/at_rule/at_rule_font_palette_values.css:1]: Thank you, you are correct.

**Review by denbezrukov — APPROVED** [review]
It looks good.
Appreciate your help!🙏🏼

**Pull request merged**